### PR TITLE
[DOC][ZEPPELIN-1198] Add a guide docs for running Spark standalone mode with Zeppelin

### PR DESCRIPTION
**Pull request by AhyoungRyu**
I wrote `spark_cluster_mode.md` based on [your issue](https://github.com/astroshim/zeppelin_backends_docker/issues/2). Please feel free to add new contents or update if it's need :) 

And here is the screenshot images. It will be shown on Zeppelin docs site like below. 
- Dropdown menu: More -> Advanced -> Zeppelin on Spark Cluster Mode(Standalone)
  screen shot 2016-07-22 at 10 48 35 am
- `spark_cluster_mode.md` docs
  screen shot 2016-07-22 at 10 47 04 am
  screen shot 2016-07-22 at 10 47 12 am

### CODE REVIEW

**Comment by astroshim** [comment]
Really great job @AhyoungRyu

**Comment by AhyoungRyu** [comment]
@astroshim I pushed more commits and updated PR description with some screenshots.

File: docs/install/spark_cluster_mode.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+You can simply set up Spark standalone environment with below steps. 
+
+> **Note :** Since Apache Zeppelin and Spark use same `8080` port for their web UI, you might need to change `zeppelin.server.port` in `conf/zeppelin-site.xml`.
+
+### 1. Build Docker file
+You can find docker script files under `scripts/docker/spark-cluster-managers`.
+
+```
+cd $ZEPPELIN_HOME/scripts/docker/spark-cluster-managers
+docker build -t "spark_standalone"
```
Comment (astroshim) [inline docs/install/spark_cluster_mode.md]: This line should be `docker build -t "spark_standalone" .` you missed '.' in the end of line. :)

**Comment by astroshim** [comment]
Following image is spark screen shot.

image

**Comment by AhyoungRyu** [comment]
@astroshim The screenshot image you wanna show is not shown properly :)

**Comment by astroshim** [comment]
Following image is for verifying that the spark is running in docker.
image

**Comment by AhyoungRyu** [comment]
I addressed all your feedback. Could you check please?

**Comment by astroshim** [comment]
@AhyoungRyu It's really good. Thank you for your great effort. :+100:

**Comment by AhyoungRyu** [comment]
Thanks. My pleasure :)

**Pull request closed without merging**